modules/interactive_music/doc_classes/AudioStreamInteractive.xml

```diff
@@ -119,7 +119,7 @@
 			<param index="0" name="from_clip" type="int" />
 			<param index="1" name="to_clip" type="int" />
 			<description>
-				Return true if a given transition exists (was added via [method add_transition]).
+				Return [code]true[/code] if a given transition exists (was added via [method add_transition]).
```

Comment:
```suggestion
				Returns [code]true[/code] if a given transition exists (was added via [method add_transition]).
```
*Extremely* nitpicky, but it seems like "Returns `true`" is favored over "Return `true`" for the start of property descriptions. Just from searching in these changes, "Returns" is used much more often, and "return `true`" is used for other purposes like "should return `true`".

There are a couple other instances if you decide to apply this suggestion

Reply:
It's not nitpicky, this is entirely appropriate. "This method" is usually implied.